subsystems/sonarSensor.py

```diff
@@ -33,7 +31,7 @@ def createChannel(self):
         sonar_ip = self.sonar_ip
         sonar_port = self.sonar_port
         try:
-            self.channel = UDPChannel(local_ip="10.10.76.2", local_port=5811, 
```

Comment:
We could be clever here and check if the sonar_ip is '127.0.0.1', and then just use that for the local_ip as well, otherwise the robot IP.  That way this works two interesting ways.